pkg/ipam/ipam_windows.go

```diff
@@ -0,0 +1,38 @@
+// +build windows
+
+// Copyright 2015 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipam
+
+import (
+	"github.com/containernetworking/cni/pkg/invoke"
+	"github.com/containernetworking/cni/pkg/types"
+	"github.com/containernetworking/cni/pkg/types/current"
+)
+
+func ExecAdd(plugin string, netconf []byte) (types.Result, error) {
```

Comment:
It looks like these functions are duplicated - Can you just move ConfigureIface to a build-specific file?

Reply:
Moved the execXX funcs to exec.go - let me know if thats not what you had in mind